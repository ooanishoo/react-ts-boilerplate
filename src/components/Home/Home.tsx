import React from 'react'

const Home: React.FC = () => {
    return <div data-testid="home">This is a Home Page</div>
}

export default Home
