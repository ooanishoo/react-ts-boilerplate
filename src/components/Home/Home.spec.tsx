import { render } from '@testing-library/react'
import React from 'react'
import Home from './Home'

describe('<Home/>', () => {
    test('should render without crashing', () => {
        const expectedOutput = 'Home'
        const { container, getByTestId } = render(<Home />)
        expect(container).toMatchSnapshot()
        expect(getByTestId('home')).toHaveTextContent(expectedOutput)
    })
})
