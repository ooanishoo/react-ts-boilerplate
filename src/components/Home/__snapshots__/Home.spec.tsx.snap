// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home/> should render without crashing 1`] = `
<div>
  <div
    data-testid="home"
  >
    This is a Home Page
  </div>
</div>
`;
